web_ui/src/pages/landing-page/landing-page-workspace/landing-page-workspace.component.tsx

```diff
@@ -45,19 +45,24 @@ export const LandingPageWorkspace = () => {
 
     return (
         <Flex direction={'column'} height={'100%'}>
-            <ProjectsActions
-                shouldShowProjectActions={hasProjects || hasNameFilter}
-                queryOptions={queryOptions}
-                setQueryOptions={setQueryOptions}
-                datasetImportDialogTrigger={datasetImportDialogTrigger}
-            />
+            {hasProjects && (
+                <ProjectsActions
+                    shouldShowProjectActions={hasProjects || hasNameFilter}
+                    queryOptions={queryOptions}
+                    setQueryOptions={setQueryOptions}
+                    datasetImportDialogTrigger={datasetImportDialogTrigger}
+                />
+            )}
 
             <Flex
                 flex={1}
                 minHeight={0}
                 direction={'column'}
                 position={'relative'}
-                UNSAFE_style={{ overflowY: 'auto' }}
+                UNSAFE_style={{
+                    overflowY: 'auto',
+                    marginTop: hasProjects ? '0px' : '32px',
```

Comment:
Can this margin be moved to the `NoProjectArea` component, so we don't need this condition?